src/table/components/TableTotals.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { memo, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import TableRow from '@mui/material/TableRow';
+import TableCell from '@mui/material/TableCell';
+import { useContextSelector, TableContext } from '../context';
+import { getHeaderStyle, getTotalsCellStyle } from '../utils/styling-utils';
+import { totalHeadHandleKeyPress } from '../utils/handle-key-press';
+import { handleClickToFocusHead } from '../utils/handle-accessibility';
+
+function TableTotals({ rootElement, tableData, theme, layout, translator, keyboard }) {
+  const { columns, paginationNeeded, totalsPosition } = tableData;
+  const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
+  const totalHeaderStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme.name()]);
+  const headRowStyle = {
+    '& :last-child': {
+      borderRight: paginationNeeded && 0,
+    },
+    'th:first-of-type': {
+      borderLeft: !paginationNeeded && '1px solid rgb(217, 217, 217)',
+    },
+  };
+
+  return (
+    <TableRow sx={headRowStyle} className="sn-table-row">
+      {columns.map((column, columnIndex) => {
+        return (
+          <TableCell
+            sx={getTotalsCellStyle(totalHeaderStyle, layout)}
+            key={column.id}
+            align={column.align}
+            className="sn-table-cell"
+            tabIndex={-1}
+            onKeyDown={(e) => {
+              let cellCoord;
+              if (totalsPosition === 'bottom') {
+                const rowIndex = rootElement.getElementsByClassName('sn-table-row').length;
+                cellCoord = [rowIndex - 1, columnIndex];
+              } else cellCoord = [1, columnIndex];
```

Comment:
the amount of rows already exist on pageInfo. it should be the `rowsPerPage` property. You can use that to simplify this:
```suggestion
              const cellCoord = [totalsPosition === 'bottom' ? rowsPerPage : 1, columnIndex];
```
Might need `rowsPerPage + 1` to  or something for the number to be correct

Reply:
yes, you are right sorry. But the information is on `tableData`. rows.length should give you the current amount of rows showing

Reply:
yeah but you know that you have one header row and one totals row, so just add those

Reply:
and this would be the same rowIdx for both `totalHandleKeyPress` and `handleClickToFocusCell`